subsys/bluetooth/host/gatt.c

```diff
@@ -1318,6 +1318,24 @@ int bt_gatt_service_unregister(struct bt_gatt_service *svc)
 
 	return 0;
 }
+
+bool bt_gatt_service_is_register(struct bt_gatt_service *svc)
+{
+	bool ret = false;
+	sys_snode_t *node;
+
+	k_sched_lock();
+	SYS_SLIST_FOR_EACH_NODE(&db, node) {
+		if (&svc->node == node) {
+			ret = true;
+			goto exit;
+		}
+	}
+
+exit:
+	k_sched_unlock();
```

Comment:
`SYS_SLIST_FOR_EACH_NODE` is basically just for loop, right? So we could probably just `break` instead of using a `goto`:

```suggestion
	SYS_SLIST_FOR_EACH_NODE(&db, node) {
		if (&svc->node == node) {
			ret = true;
			break;
		}
	}

	k_sched_unlock();
```